src/Uno.UWP/ApplicationModel/Contacts/ContactQuerySearchFields.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	[Flags]
+	public enum ContactQuerySearchFields

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the 'FlagsAttribute' from this enum.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4900834)

Reply:
Can codacy be configured to not report this false issue every time I change something in this PR?
Yes, this enum have to be [Flags], and yes, -1 means all "1".